modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/css/main.scss

```diff
@@ -327,6 +327,10 @@
 	width: 100% !important;
 }
 
+.portal-popup .notifications-cell-editor {
+	top: calc(239px - 61px) !important;
```

Comment:
@naraandrade is this calc necessary?

Reply:
@albinojunior I thought it would be better to explain that I am removing a space relative to the modal, but it is not really necessary.

Reply:
I'll apply the number directly, thank you :)